### ISSUE TITLE
Can the trivy offline db be included in the installer?

### ISSUE THREAD

**Issue by clannon** [goharbor/harbor#18204]
I tested the newest version 2.7.0 and wanted to use the scan feature --with-trivy, but it did not worked as i expected. For some reason, the net speed is very low in my country to update the db and the result got an error because of timeout download, so I set "skip_update: true", but it still showed "--skip-update cannot be specified on the first run". I followed the instruction("Trivy configuration" part in harbor.yml) to download the offline db and dropped it to the volume, and still got that error because the trivy is already 0.35 and the offline db is out of date and incompatible. After many searches, I had to "docker exec -it" into the trivy-adapter-photon container and executed "trivy image --download-db-only" to get the db. After half an hour, the scanner finally worked and showed the report.

![1](https://user-images.githubusercontent.com/34529891/218265610-70dfd971-1241-4c34-98e9-207bd211bc5a.png)

<img width="1266" alt="2" src="https://user-images.githubusercontent.com/34529891/218265593-cf2617df-813b-497d-8447-16a7a44fd072.png">

So, can the offline db be added into the installer as a initial part of this feature when the new harbor version is published, so that it can be used after the installation immediately? It is very important for the low net speed  situations like mine. Thanks.

**Comment by MinerYang** [comment]
Please refer to this https://github.com/goharbor/harbor/wiki/Harbor-FAQs#trivy-in-air-gapped-environment
cc @zyyw help to confirm.

**Comment by clannon** [comment]
> Please refer to this https://github.com/goharbor/harbor/wiki/Harbor-FAQs#trivy-in-air-gapped-environment cc @zyyw help to confirm.

I really refered the guide when I tested above. What I said means the document is OUT OF DATE. The url in the "Air-Gapped Environment" is 404 for at least 2 weeks from I posted this issue. There is no other way for the offline trivy db, unless getting into the container to download for 30 minutes in my low net speed situation. So I suggested why not put a latest or trusted trivy db into the offline installer, so that we can get rid of downloading db with "skip_update=true" with a better out-of-the-box security experience.
